add companies logo that support the sigstore development (#120)
sigstore/community: README.md

## Adding Users/Collaborators, Teams, Repositories to Sigstore GitHub Organization

To add or change to add/change an Org Member, team, collaborator or repository please [follow this steps](github-sync/README.md)

## Support Tools

> We would like to thank the following companies that support the sigstore development by giving some OpenSource plan to we use.

Reviewer: ```suggestion
We would like to thank the following companies who provide us with an OpenSource plan to support Sigstore development.
```

Author: thanks
